connector/connect/common/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -93,6 +93,9 @@ message Unknown {}
 message RelationCommon {
   // (Required) Shared relation metadata.
   string source_info = 1;
+
+  // (Optional) A globally unique id for a given connect plan.
```

Comment:
shall we mention it's per-client global?

Reply:
sure